OpenMTR/code/DebugUtils.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OpenMTR
+{
+    class DebugUtils
+    {
+        /// <summary>
```

Comment:
I like the documentation but I feel like it can be simplified. What about doing something like this:

```c#
//This method writes a message to the console for easier debugging
``` 

This lets the developer know what the method does at a high level without some bloat the auto-commenter attaches on top of the method signature.